src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -20,99 +20,15 @@ interface Props {
 }
 
 class Filter extends Component<Props> {
-  renderCurrentlySelected(filter, state) {
```

Comment:
The code deletions here were leftover from an earlier version of the filter, they're not used anymore.